wgsl/index.bs

```diff
@@ -9805,8 +9814,8 @@ For each [=formal parameter=] of a function, one or two tags are computed:
   <thead>
     <tr><th>Parameter Return Tag<th>Description
   </thead>
-  <tr><td><dfn noexport>ParameterReturnRequiredToBeUniform</dfn>
-      <td>The parameter [=shader-creation error|must=] be a [=uniform value=] in order for the [=return value=] to be a uniform value.
+  <tr><td><dfn noexport>ParameterReturnContentsRequiredToBeUniform</dfn>
+      <td>The parameter [=shader-creation error|must=] be a [=uniform value=] in order for the [=return value=] contents to be a uniform value.
```

Comment:
I think the "contents" is in the wrong place (should relate to the parameter, not the return value). This should actually be "the parameter contents must be a uniform value in order for the return value to be a uniform value".

Reply:
Would you change the description too?

> The parameter must be a uniform value in order for the return value to be a uniform value.
If the parameter is a pointer, then additionally value stored in the memory pointed to by the parameter must be uniform in order for the return value to be uniform.

Reply:
Yes, my comment was more about the description than the tag name itself.

FWIW Tint uses two separate tags for pointer value vs pointer contents here, which is not necessary for V1 but will be required when we remove restrictions on pointer parameters.